File: .tmp_py4013/batch_11_dump/3274_check_if_two_chessboard_squares_have_the_same_color.js
// LeetCode 3274 - Check if Two Chessboard Squares Have the Same Color
// https://leetcode.com/problems/check-if-two-chessboard-squares-have-the-same-color/

var checkTwoChessboards = function(coordinate1, coordinate2) {
    const c1 = (coordinate1.charCodeAt(0) - 97) + (coordinate1.charCodeAt(1) - 49);
    const c2 = (coordinate2.charCodeAt(0) - 97) + (coordinate2.charCodeAt(1) - 49);
    return c1 % 2 === c2 % 2;
};

File: .tmp_rs4013/rust_check15/3274_check_if_two_chessboard_squares_have_the_same_color.rs
struct Solution;
// LeetCode 3274 - Check if Two Chessboard Squares Have the Same Color
// https://leetcode.com/problems/check-if-two-chessboard-squares-have-the-same-color/

impl Solution {
    pub fn check_two_chessboards(coordinate1: String, coordinate2: String) -> bool {
        let b1 = coordinate1.as_bytes();
        let b2 = coordinate2.as_bytes();
        let c1 = (b1[0] - b'a') as i32 + (b1[1] - b'1') as i32;
        let c2 = (b2[0] - b'a') as i32 + (b2[1] - b'1') as i32;
        c1 % 2 == c2 % 2
    }
}

fn main() {}
